platforms/access-bridge/src/app-config.ts

```diff
@@ -0,0 +1,46 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import * as dotenv from 'dotenv';
+
+// Load environment variables from .env files
+function loadEnvFiles() {
```

Comment:
I'd suggest we use vite to inject the (non-secret) env variables directly into code and a secret manager to set env vars for the secrets: https://cloud.google.com/run/docs/configuring/services/secrets